_posts/en/pages/faq/2016-01-21-rbf-faq.md

```diff
@@ -0,0 +1,237 @@
+---
+### orinally sourced from https://www.reddit.com/r/Bitcoin/comments/3urm8o/optin_rbf_is_misunderstood_ask_questions_about_it/
+title: Opt-in RBF FAQ
+name: optin-rbf-faq
+type: page
+permalink: /en/faq/optin_rbf
+share: true
+---
+{% include _toc.html %}
+
+##What is RBF?
+
+RPF or replace by fee is a way of enabling pending transactions to be modified -- or double spent.
+
+Opt-in RBF means transactions can be marked double spendable or not.
+
+##Does RBF implementation change the likelihood that a "non-RBF" (max value nSequence) transaction can be double-spent?
+
+No.  A transaction must be marked replaceable (sequence number *below* MAX*-1*) to be replaced.
```

Comment:
I'm not a native English speaker, but I think this should be "to be replaceable", instead of "to be replaced".